.eslintrc.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
-  parser: 'babel-eslint',
+  parser: '@babel/eslint-parser',
   parserOptions: {
+    requireConfigFile: false,
```

Comment:
See https://github.com/babel/babel/tree/main/eslint/babel-eslint-parser#additional-parser-configuration

According to https://github.com/babel/babel-eslint#when-should-i-use-babel-eslint it should behave the same.